### PR TITLE
fix(react-ui-kit): Set default Avatar & AvatarGrid size

### PR DESCRIPTION
**Pull request by Yserz**
Closes #2128

### CODE REVIEW

File: packages/react-ui-kit/src/Identity/AvatarGrid.tsx [hunk 1]
```diff
@@ -46,24 +47,25 @@ const avatarGridStyle: <T>(props: Props<T>) => ObjectInterpolation<undefined> =
...
   };
 };
 
 const filteredAvatarGridProps = (props: Props) =>
-  filterProps(props, ['backgroundColor', 'borderColor', 'items', 'size']);
+  filterProps(props, ['backgroundColor', 'borderColor', 'items', 'size', 'borderWidth']);
 
-export const AvatarGrid = (props: Props) => {
-  const {size, items} = props;
+export const AvatarGrid = ({borderWidth = 1, size = DEFAULT_AVATAR_SIZE + borderWidth * 2, items, ...props}: Props) => {
```
Comment (lipis) [inline packages/react-ui-kit/src/Identity/AvatarGrid.tsx]: I think to be perfect.. we need to subtract the borderWidth * 2
```suggestion
export const AvatarGrid = ({borderWidth = 1, size = DEFAULT_AVATAR_SIZE - borderWidth * 2, items, ...props}: Props) => {
```

File: packages/react-ui-kit/src/Identity/AvatarGrid.tsx [hunk 2]
```diff
@@ -75,10 +77,16 @@ export const AvatarGrid = (props: Props) => {
...
-            size={size / 2}
+            size={size / 2 - borderWidth}
           />
         ) : (
-          <div css={{backgroundColor: COLOR.GRAY_DARKEN_80, width: size / 2, height: size / 2}} />
+          <div
+            css={{
+              backgroundColor: COLOR.GRAY_DARKEN_80,
+              height: size / 2 - borderWidth,
+              width: size / 2 - borderWidth,
```
Comment (lipis) [inline packages/react-ui-kit/src/Identity/AvatarGrid.tsx]: not sure about here now.. need to investigate..!

File: packages/react-ui-kit/src/Identity/AvatarGrid.tsx [hunk 3]
```diff
@@ -75,10 +77,16 @@ export const AvatarGrid = (props: Props) => {
...
             name={item.name}
-            size={size / 2}
+            size={size / 2 - borderWidth}
           />
         ) : (
-          <div css={{backgroundColor: COLOR.GRAY_DARKEN_80, width: size / 2, height: size / 2}} />
+          <div
+            css={{
+              backgroundColor: COLOR.GRAY_DARKEN_80,
+              height: size / 2 - borderWidth,
```
Comment (lipis) [inline packages/react-ui-kit/src/Identity/AvatarGrid.tsx]:
```suggestion
              height: size / 2,
```

File: packages/react-ui-kit/src/Identity/AvatarGrid.tsx [hunk 4]
```diff
@@ -75,10 +77,16 @@ export const AvatarGrid = (props: Props) => {
             forceInitials={item.forceInitials}
             isAvatarGridItem
             name={item.name}
-            size={size / 2}
+            size={size / 2 - borderWidth}
```
Comment (lipis) [inline packages/react-ui-kit/src/Identity/AvatarGrid.tsx]:
```suggestion
            size={size / 2}
```

**Comment by Yserz** [comment]
nope
image

**Pull request merged**